samples/tfm/provisioning_image/README.rst

```diff
@@ -0,0 +1,85 @@
+.. _provisioning_image:
+
+Provisioning image
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to initialize the provisioning process of a Trusted Firmware M(TF-M) image.
+The provioning process sets the device in the Provisioning Platform Root of Trust(PRoT) Security Lifecycle as defined in the security model of Platform Security Architecture(PSA).
+This sample does not include a TF-M image, it is a Zephyr image which is required to run before the TF-M image in order to allow the provisioning of the TF-M image later.
```

Comment:
```suggestion
This sample, when executed, will provision the device in a manner compatible with Trusted Firmware M (TF-M).
The device will after execution be in the "PRoT Provisioning" Security Lifecycle state as defined in the "Platform Security Model" of the Platform Security Architecture(PSA).
This sample does not include a TF-M image, it is a Zephyr image which is intended to be flashed, run, and then erased again before the TF-M image is flashed.
```

Reply:
"in order to allow the provisioning of the TF-M image later"

@Vge0rge : Do we do any provisioning in TF-M later?

Reply:
Answered my own question. We don't do any provisioning in TF-M. So I think it's misleading to say "in order to allow the provisioning of the TF-M image later".

Reply:
Hm, the transitioning between the provisioning state -> secure state is happening in TF-M. So I cannot say that there is no provisioning happening in TF-M. We basically split the provisioning process, as it is stated in the PSA spec in two steps. The first step is this image and the second step is the TF-M image nrf_provisioning.c logic.